### ISSUE TITLE
Investigate middle click to paste

### ISSUE THREAD

**Issue by darrenburns** [Textualize/textual#5319]
On the Posting repo, there's been a couple of Linux and Windows users claim the pasting isn't working.

Turns out they're using middle click to try to paste, rather than a keybind.

We should investigate to see if we can make middle click paste work.

**Comment by tzot** [comment]
Yes please. Or even allowing Shift+Insert to paste the PRIMARY selection — unless Shift+Ins is taken by paste from the CLIPBOARD selection…
